src/components/header.svelte

```diff
@@ -64,14 +74,17 @@
                             </li>{/if}
                     {/each}
                 </ul>
+                <!-- Burger Icon -- Mobile  -->
+                <Burger bind:open={sideMenuOpen} onClick={toggleMenu} {sideMenuOpen} classes="flex my-auto lg:hidden" />
+            {/if}
+        </div>
+    </div>
+</nav>
 
-                <!-- Mobile -->
-                <div class="flex pt-3 lg:hidden">
-                    <Burger bind:open={sideMenuOpen} onClick={toggleMenu} {sideMenuOpen} />
-                </div>
-                <aside class="bg-green-200 h-screen w-0 absolute left-0 top-0 lg:hidden {sideMenuOpen ? 'open' : ''}">
+<!-- Mobile Menu   -->
+ <aside class="bg-green-200 h-screen w-0 fixed left-0 top-0 lg:hidden z-40 whitespace-nowrap {sideMenuOpen ? 'open' : ''}">
```

Comment:
Something is happening here:
https://www.awesomescreenshot.com/video/9445992?key=1adb0cb34398fc632714eaef99dcfbbb